packages/jest-jasmine2/package.json

```diff
@@ -18,6 +18,7 @@
     "jest-matcher-utils": "^22.2.0",
     "jest-message-util": "^22.2.0",
     "jest-snapshot": "^22.2.0",
+    "source-map": "^0.6.0",
```

Comment:
newest is 0.7.1, but they changed their API to be asynchronous, so we can't use it